config/console.ini

```diff
@@ -4,16 +4,40 @@
 
 ### CONSOLE OPTIONS ###
 
+## EnableLogRotation (bool)
```

Comment:
That's an interesting thought. To me, `Console` is an interface for "I want something to appear on stdout". It's a nice set of convenience functions for controlling all stdout/stderr messages. I think we're agreed on that point.

Say I'm running etheos as a systemctl daemon (which is how `eoserv.moffat.io` runs it) or in the cloud (which is how the dev/test instances are run). Now, I want my stdout to be redirected to a file. So I set `LogOut` and `LogErr` accordingly. I think we're agreed on this point too.

Now, I've been running a server for a couple months and my log files are _huge_. I want to rotate log files so they're split up a little more conveniently, and have etheos automatically manage old logs and remove them. That's what this PR addresses. I think we're agreed that this is a good thing to have.

Now - to your point about `Logger` vs `Console`, I think there is a distinction here, but I think the distinction I'd like to make is that `Console` will always go to the default and `Logger` is a special type of logging, such as the examples you listed. "The default" meaning whatever has been set up in `LogOut`/`LogErr` or the log rotation properties. I think introducing a Logger abstraction makes sense as a middle layer between `Console` and where the output eventually ends up, but not as part of this PR.

I do eventually want to add structured logging. I think `Logger` could take a "provider" of sorts, where we have one or more providers set up in `Logger.ini`. We could conceivably set up the Console logging as the default through this system too so they're all set up the same way. Then we could have a special structured logger (if configured) for certain types of messages, an admin command logger, a trade logger, etc. But like I said - I don't want to get into that just yet.

In the end that would mean `Console` isn't limited to "log to stdout/stderr", it's "log to the default provider"; whether that is stdout or a file or structured log system is up to the configuration. I'd prefer to keep that "default" since it's already the assumption in every output call in the code base, if we have special cases we can use a different logger for those.